From 239fd26d32ac5a420996e1b6d4d1e4824daf95c5 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Fri, 13 Nov 2020 21:07:54 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3568: Change CLK_TSADC_TSEN to 17MHz

Make the bandgap clock close to its typical frequency 16MHz.

Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
Change-Id: I60b98438a89ab91fb9e2ac009ad8e2b2cba4a5a3
---
 arch/arm64/boot/dts/rockchip/rk3568.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3568.dtsi b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
index 13734d80dcd0..3350bd77b174 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
@@ -2291,7 +2291,7 @@
 		clocks = <&cru CLK_TSADC>, <&cru PCLK_TSADC>;
 		clock-names = "tsadc", "apb_pclk";
 		assigned-clocks = <&cru CLK_TSADC_TSEN>, <&cru CLK_TSADC>;
-		assigned-clock-rates = <20000000>, <700000>;
+		assigned-clock-rates = <17000000>, <700000>;
 		resets = <&cru SRST_TSADC>, <&cru SRST_P_TSADC>,
 			 <&cru SRST_TSADCPHY>;
 		reset-names = "tsadc", "tsadc-apb", "tsadc-phy";
-- 
2.35.3

